### ISSUE TITLE
Plex image options

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockon-registry#363]
We have for some time now favoured the linuxserver.io images. However we also like to use official docker images where available.

For Plex this is now: https://hub.docker.com/r/plexinc/pms-docker/

Where-as our current Plex rock-on uses: https://hub.docker.com/r/linuxserver/plex

A major regression if we switched to the official would be the loss of ARM64 compatibility. Which is not desirable.

**Comment by Hooverdan96** [comment]
I agree with the official docker image usage. However, especially with something like Plex I feel the linuxserver images are a more convenient way of managing the two key things: access and data directory ownership (the easy user/group id management is one of the things that makes linuxserver  great for more casual users on a linux platform in my opinion).

Considering that we would lose the arm64 compatibility I think that's a showstopper. I know enough people that still use their Raspberry Pi to serve up Plex content (not necessarily based on Rockstor, but I would assume, that there is a large enough group out there on Rockstor that is using it).
While not a good idea from a privacy standpoint, it would of course be great if we knew which "platform" Rockstor and Rockons are being run on to see whether that is/is not a concern.

In general, I feel the linuxserver.io images are a good compromise in terms of usability and being maintained near "real-time" with security and base image updates - especially the images that auto-update inside the container from the latest releases of a given application when restarted.

**Comment by phillxnet** [comment]
@Hooverdan96 Agreed on the ARM64 point, I was disapointed to see that not (hopefully only yet) available on the official side.

Re: user group setting, we do have these environmental variables (-e ) in the official image:

> PLEX_UID The user id of the plex user created inside the container.
> PLEX_GID The group id of the plex group created inside the container

But all in I'm currently siding with leaving things as they are currently. Just wanted to note this by way of an issue to ensure we are aware of the current state of this official image.

@FroggyFlox for a consensus, I'm leaning to continuing with our linuxserver/plex image. And given @Hooverdan96's:

> In general, I feel the linuxserver.io images are a good compromise

We may just want to close this issue, as I just wanted to re-examine our position on this one.

**Comment by Hooverdan96** [comment]
For now, I will close this issue. We can, as always revisit over time to see whether the situation has changed then.
